deploy/stacks/lambda_api.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         prod_sizing=False,
         user_pool=None,
         pivot_role_name=None,
+        reauth_ttl=5,
```

Comment:
In the cdk.json it will be an int right? Then we will define it in the env variables as string and then back to int and datetime delta in api_handler

Reply:
I agree! In the cdk.template can you add the info that it is INT and in minutes?